apisix/schema_def.lua

```diff
@@ -491,6 +491,15 @@ _M.route = {
             minItems = 1,
             uniqueItems = true,
         },
+        upstream_timeout = {
```

Comment:
Just use `timeout` would seem a better name. It would override upstream timeout but it's still for routes but not upstreams